modules/openapi-generator/src/main/resources/python/model_templates/methods_shared.mustache

```diff
@@ -27,7 +27,7 @@
         if self.get("_spec_property_naming", False):
             new_inst = cls._new_from_openapi_data()
         else:
-            new_inst = cls.__new__(cls)
+            new_inst = cls.__new__(cls, **self.__dict__)
```

Comment:
But line 33 below assigns properties
Why is this needed?

Reply:
Hi spacether, 

I don't know if this solution is the best. But without this change the example/test doesn't work.
```
>>> from copy import deepcopy
>>> from petstore_api.model.dog import Dog
>>> model_dog = Dog(class_name='Dog', color='white', breed='Jack Russel Terrier')
>>> new_dog = deepcopy(model_dog)

petstore_api.exceptions.ApiValueError: Cannot deserialize input data due to missing discriminator. The discriminator property 'className' is missing at path: ()
```
The error occurs inside the deepcopy loop, when running the `new_inst = cls.__new__(cls)` operation for one of the composed schemas (`<class 'petstore_api.model.animal.Animal'>` class). In this step of the loop, the `discriminator` value is `{'class_name': {'Cat': <class 'petstore_api.model.cat.Cat'>, 'Dog': <class 'petstore_api.model.dog.Dog'>}}`, and without kargs to escape in the `cls` in `kwargs.get('_visited_composed_classes', ())` (L-231-235 model_utils.py) the deepcopy fail in L-266 (model_utils.py).

Reply:
hi @spacether! 

Do I need to provide more info?

Thanks!

Reply:
Nope no need, th